content/blog/software-metrics-workshop-report.md

```diff
@@ -0,0 +1,284 @@
+---
+title: "Report from URSSI workshop on software credit, citation, and metrics"
+date: 2019-03-24T15:17:22-07:00
+author: "Karthik Ram"
+---
+
+
+The mission of the URSSI conceptualization is to understand the challenges that researchers face regarding sustainable research software. One particular challenge identified in our first workshop was the lack of consensus on credit for research software. There is a need to gain a deeper understanding of the concept of credit for research software and how, as a community, we could define and measure impact. To explore this topic is greater detail we conducted a [topic focused workshop](https://github.com/si2-urssi/software-credit#software-metrics-credit-and-citation-workshop) on software metrics and credit at the [National Center for Ecological Analysis and Synthesis (NCEAS)](https://www.nceas.ucsb.edu/) in Santa Barbara on January 23-24, 2019.  We invited [19 experts](https://github.com/si2-urssi/software-credit#confirmed-participants) to participate in a series of discussions and deliver short presentations on their organization's efforts in this space. Below is a summary of our discussions from the two days.
```

Comment:
AFTER
The mission of the URSSI conceptualization is to understand the challenges that researchers face regarding sustainable research software
ADD: and to design a potential institute that would address these challenges.